src/main/java/util/Http.java

```diff
@@ -88,10 +93,22 @@ public HttpResponse makeStandardRequest(HttpRequestBase request) throws IOExcept
         return executeRequest(client, request);
     }
 
+    public HttpResponse makeStandardRequest(HttpUriRequest request) throws IOException {
+        HttpClient client = HttpClientBuilder
+                .create()
+                .disableRedirectHandling()
+                .build();
+        return executeRequest(client, request);
+    }
+
     public HttpResponse executeRequest(HttpClient client, HttpRequestBase request) throws IOException {
         return client.execute(request);
     }
 
+    public HttpResponse executeRequest(HttpClient client, HttpUriRequest request) throws IOException {
```

Comment:
According to The Googs, `HttpRequestBase` implements `HttpUriRequest`, so I'm not sure you need to duplicate these methods. Just change the second parameter to be a `HttpUriRequest` instead of adding a new one.